docs/Broadcasting.md

```diff
@@ -0,0 +1,66 @@
+# Broadcasting in ONNX
+
+In ONNX, element-wise operators can take inputs with different shape,
+as long as the input tensors are broadcastable to the same shape.
+ONNX supports two types of broadcasting: multidirectional broadcasting and
+unidirectional broadcasting. We will introduce these two types of broadcasting
+respecitively in the following sections.
+
+
+## Multidirectional Broadcasting
+
+In ONNX, a set of tensors are multidirectional broadcastable to the same shape
+if one of the followig is true:
+- The tensors all have exactly the same shape.
+- The tensors all have the same number of dimensions and the length of
+each dimensions is either a common length or 1.
```

Comment:
In numpy, if the dimension value is not 1, then they should be the same, right? (common length means same length here?).

Reply:
yes